.travis.yml

```diff
@@ -2,11 +2,11 @@ language: python
 python: 3.6
 cache: pip
 
-
-before_install:
-- pip install -U pre-commit
-- pip install django==1.11
-
-script:
-- pre-commit run --all-files
-- python ./manage.py test
+jobs:
+  include:
+    - stage: pre-commit tests
+      script:
+        - pre-commit run --all-files
+    - stage: django tests
```

Comment:
You don't have to move main tests to jobs

Reply:
```
script:
- python ./manage.py test

jobs:
  include:
    - stage: pre-commit tests
      script:
      - pre-commit run --all-files
```
Like this? Or how? I do not quite understand, @webknjaz .

Reply:
Yep